financial-connections/src/main/java/com/stripe/android/financialconnections/FinancialConnectionsSheetViewModel.kt

```diff
@@ -123,7 +123,7 @@ internal class FinancialConnectionsSheetViewModel @Inject constructor(
                 val attestationInitResult = prepareStandardRequestManager()
                 val syncResponse = getOrFetchSync(
                     refetchCondition = Always,
-                    attestationInitialized = attestationInitResult.initialized
+                    supportsAppVerification = attestationInitResult.initialized
```

Comment:
Can we also rename the property on `attestationInitResult`?

Reply:
https://github.com/stripe/stripe-android/pull/9966/commits/4f9b77dbb0fdbc99c1854bd320b4b3370bf75698